prov/efa/src/rdm/rxr_env.c

```diff
@@ -269,7 +269,8 @@ void rxr_env_initialize()
  */
 int rxr_env_get_use_device_rdma()
 {
-	int ret, use_device_rdma;
+	/* Disable device RDMA by default */
+	int ret, use_device_rdma = 0;
```

Comment:
Based on my discussion with @wzamazon, `use_device_rdma` should not be set to default value of 0 always. The value of this variable depends on conditions defined in `rxr_env_define_use_device_rdma()`